Roommatch/Roommatch/ViewControllers/LoginViewController.m

```diff
@@ -0,0 +1,79 @@
+//
+//  LoginViewController.m
+//  Roommatch
+//
+//  Created by Lily Yang on 7/7/22.
+//
+
+#import "LoginViewController.h"
+#import "Lib.h"
+#import "User.h"
+
+@interface LoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *usernameTextField;
+@property (weak, nonatomic) IBOutlet UITextField *passwordTextField;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+
+- (IBAction)signUp:(id)sender {
```

Comment:
If you're not using `sender` it's okay to remove that parameter.

Reply:
Tried removing it to just be - (IBAction) signUp {}, but ran into runtime errors. Not sure if it is required syntax for button IBAction outlets. I also reconnected outlets just in case, but still had an error!